core/data-resolver/ReviewsService.ts

```diff
@@ -0,0 +1,24 @@
+import { DataResolver } from './types/DataResolver';
+import { TaskQueue } from '@vue-storefront/core/lib/sync'
+import Task from '@vue-storefront/core/lib/sync/types/Task'
+import { processLocalizedURLAddress } from '@vue-storefront/core/helpers'
+import config from 'config'
+import Review from 'core/modules/review/types/Review';
+
+const createReview = (review: Review): Promise<Task> =>
+  TaskQueue.execute({
+    url: processLocalizedURLAddress(config.reviews.create_endpoint),
+    payload: {
+      method: 'POST',
+      mode: 'cors',
+      headers: {
+        'Accept': 'application/json, text/plain, */*',
+        'Content-Type': 'application/json'
+      },
+      body: JSON.stringify({ review })
+    }
+  })
+
+export const ReviewsService: DataResolver.ReviewsService = {
+  createReview
```

Comment:
getReviewsFor(product_id) might be cool here - like getCategories In the category data resolver

Reply:
Are you sure about that? because the endpoint is even called `create_review`

Reply:
I meant as an additional method in the Service - to have two: one to create a review, second to list the reviews :P

Reply:
OK, you’re right; this helper does exactly that thing